src/components/ui/category-tab/category-tab.module.css

```
@@ -0,0 +1,93 @@
+  border: 1px solid
+    light-dark(var(--mantine-color-gray-2), var(--mantine-color-dark-4));
+  background-color: light-dark(
+    var(--mantine-color-white),
+    var(--mantine-color-dark-6)
+  );
+
+  &:first-of-type {
+    border-radius: 4px 0 0 4px;
+
+    @mixin rtl {
+      border-radius: 0 4px 4px 0;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

border-radius is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)